recipes/easydev/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.9.31" %}
+{% set name = "easydev" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/e8/6a/edeff86ca5fe5250d96523d4bc2d886b2c37a3ef4341f22ab4e2b270d121/{{ name }}-{{ version }}.tar.gz
+  md5: 68353d68eb7249269c7f8b1033e56ac3
+
+build:
+  entry_points:
+    - easydev_buildPackage=easydev.package:buildPackage
+    - multigit=easydev.multigit:main_func
+    - browse=easydev.browser:main
+    - ibrowse=easydev.browser:main
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - colorama
+    - pexpect
+
+  run:
+    - python
+    - setuptools
+    - colorama
+    - pexpect
+
+test:
+  imports:
+    - easydev
+    - easydev.progressbar
+
+about:
+  home: 'http://packages.python.org/easydev/'
+  license: LGPL-3.0
```

Comment:
Please ask upstream to include a license file and also please ask that it be included in `MANIFEST.in`. This is important so we can include the license file in the `conda` package and distribute it to users.